lib/Doctrine/ORM/Query/SqlWalker.php

```diff
@@ -854,9 +856,41 @@ public function walkIdentificationVariableDeclaration($identificationVariableDec
      */
     public function walkIndexBy($indexBy)
     {
-        $pathExpression = $indexBy->simpleStateFieldPathExpression;
```

Comment:
Why did you rename this variable?

Reply:
To answer all three of your comments, the switch statement here was simply copied from walkPathExpression(). I renamed this variable only because it was easier than changing it throughout the switch.